addons/l10n_se/data/account.account.template.csv

```diff
@@ -1,325 +1,328 @@
+a1490,1490,Övriga lagertillgångar,account.data_account_type_prepayments,l10nse_chart_template,l10n_se.account_tag_4,False
+a1510,1510,Kundfordringar,account.data_account_type_receivable,l10nse_chart_template,l10n_se.account_tag_5,True
+a1513,1513,Kundfordringar - delad faktura,account.data_account_type_receivable,l10nse_chart_template,l10n_se.account_tag_5,True
+a1519,1519,Nedskrivning av kundfordringar,account.data_account_type_receivable,l10nse_chart_template,l10n_se.account_tag_5,True
+a1580,1580,Fordringar för kontokort och kuponger,account.data_account_type_receivable,l10nse_chart_template,l10n_se.account_tag_5,True
+a1610,1610,Kortfristiga fordringar hos anställda,account.data_account_type_receivable,l10nse_chart_template,l10n_se.account_tag_5,True
+a1630,1630,Avräkning för skatter och avgifter (skattekonto),account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1640,1640,Skattefordringar,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1650,1650,Momsfordran,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1680,1680,Andra kortfristiga fordringar,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1710,1710,Förutbetalda hyreskostnader,account.data_account_type_prepayments,l10nse_chart_template,l10n_se.account_tag_5,False
+a1720,1720,"Förutbetalda leasingavgifter, kortfristig del",account.data_account_type_prepayments,l10nse_chart_template,l10n_se.account_tag_5,False
+a1730,1730,Förutbetalda försäkringspremier,account.data_account_type_prepayments,l10nse_chart_template,l10n_se.account_tag_5,False
+a1740,1740,Förutbetalda räntekostnader,account.data_account_type_prepayments,l10nse_chart_template,l10n_se.account_tag_5,False
+a1750,1750,Upplupna hyresintäkter,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1760,1760,Upplupna ränteintäkter,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1790,1790,Övriga förutbetalda kostnader och upplupna intäkter,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1810,1810,Andel i börsnoterade företag,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_6,False
+a1880,1880,Andra kortfristiga placeringar,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_6,False
+a1890,1890,Nedskrivning av kortfristiga placeringar,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_6,False
+a1910,1910,Kassa,account.data_account_type_liquidity,l10nse_chart_template,l10n_se.account_tag_7,True
+a1920,1920,PlusGiro,account.data_account_type_liquidity,l10nse_chart_template,l10n_se.account_tag_7,True
+a1930,1930,Företagskonto/checkkonto/affärskonto,account.data_account_type_liquidity,l10nse_chart_template,l10n_se.account_tag_7,True
```

Comment:
1910, 1920, 1930: liquidity +allow reconciliation = dubious. IMO these accounts should be simply removed. @tsb-odoo what do you think?

Reply:
Those accounts seems to be used a lot in the reports examples/samples I found, and those lines were not added by me (just the tags) but on the commit [a0f0cdd7](https://github.com/odoo-dev/odoo/commit/a0f0cdd70e88e9da322b9e02157c58e233ba7104)
I can change the reconciliation if needed